main.go

```diff
@@ -14,12 +17,27 @@ func main() {
 	)
 	flag.Parse()
 
-	log.Printf("Starting on %s", *port)
-	log.Fatal(http.ListenAndServe(":"+*port, http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		log.Printf("%s - %s", r.Method, r.URL)
-		w.WriteHeader(200)
-		io.WriteString(w, "Ok\n")
-	})))
+	if len(os.Args) < 2 {
```

Comment:
use flag.Args here instead so you don't have to worry about the order quiet as much